### ISSUE TITLE
Tests are failing due to web workers

### ISSUE THREAD

**Issue by nop33** [alephium/desktop-wallet#453]
In the #444 the tests are failing because `ReferenceError: Worker is not defined`. I've installed `@vitest/web-worker` https://www.npmjs.com/package/@vitest/web-worker, but now I get a different error:

```
TypeError: runner.resolveUrl is not a function
 ❯ new _a node_modules/@vitest/web-worker/dist/pure.js:398:14
 ❯ src/contexts/addresses.tsx:35:42
     33| import { useGlobalContext } from './global'
     34|
     35| const deriveAddressesFromIndexesWorker = new Worker(
       |                                          ^
     36|   new URL('../workers/deriveAddressesFromIndexes.ts', import.meta.url)
     37| )
 ❯ VitestRunner.directRequest node_modules/vitest/dist/chunk-vite-node-source-map.d0aa773d.js:306:5
```

I looked at the source code of `@vitest/web-worker` and it seems that `resolveUrl` is there...
I tried to reproduce a minimal example, but everything seems to work fine: https://stackblitz.com/edit/vitest-dev-vitest-v2nnta?file=test/basic.test.ts

**Comment by nop33** [comment]
After upgrading `vitest` to the latest version (`v0.26.2`), the tests now just get stuck:

```
 ✓ src/tests/settings.test.ts (6)
 ✓ src/tests/links.test.ts (1)
 ✓ src/tests/addresses.test.ts (1)
 ✓ src/tests/migration.test.ts (3)
 · src/tests/pages/HomePage.test.tsx (5)
 · src/tests/App.test.tsx (3)
```

**Comment by nop33** [comment]
Fixed in #444
